handlebars/express-handlebars.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const Path = require('path');
```

Comment:
```suggestion
const path = require('path');
```

Convention is to use lowercase for non-class modules. Think there are a few instances of that in this PR that I would suggest changing.